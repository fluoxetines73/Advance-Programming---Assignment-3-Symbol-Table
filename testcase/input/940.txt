RPRINT
RPRINT
RPRINT
PRINT
INSERT a string
BEGIN
INSERT c number random
LOOKUP c
END
RPRINT
PRINT
ASSIGN a 'ab c
INSERT e_123_2r number
ASSIGN e_123_2r 1001
INSERT y number
ASSIGN x unimportant