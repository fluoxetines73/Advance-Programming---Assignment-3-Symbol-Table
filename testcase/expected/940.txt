Invalid: INSERT c number random